### PR TITLE
Add support for the -target arg

### PR DESCRIPTION
**Pull request by mikebrooks-ovo**

### CODE REVIEW

File: terraform/orb.yml [hunk 1]
```diff
@@ -215,7 +219,7 @@ commands:
               exec terraform workspace new "$workspace" "$module_path"
             else
               echo "$workspace already exists"
-            fi 
+            fi
```
Comment (danielflookovo) [inline terraform/orb.yml]: 👀

Comment (mikebrooks-ovo) [reply terraform/orb.yml]: "format on save" :D

**Comment by danielflookovo** [comment]
👍 Can you add it for the plan and check jobs/commands, and update the readme

File: terraform/apply.sh [hunk 2]
```diff
@@ -13,6 +13,12 @@ EOF
 
 set +e
 
+if [ "<< parameters.auto_approve >>" = "true" && -n "<< parameters.target >>" ]; then
```
Comment (danielflookovo) [inline terraform/apply.sh]: I don't think thats valid bash
```suggestion
if [[ "<< parameters.auto_approve >>" = "true" && -n "<< parameters.target >>" ]]; then
```

**Pull request merged**